README.md

```diff
@@ -1,3 +1,517 @@
+[![CircleCI Status](https://img.shields.io/circleci/project/github/react-native-community/react-native-datetimepicker/master.svg)](https://circleci.com/gh/react-native-community/workflows/react-native-datetimepicker/tree/master) ![Supports Android and iOS](https://img.shields.io/badge/platforms-android%20|%20ios-lightgrey.svg) ![MIT License](https://img.shields.io/npm/l/@react-native-community/slider.svg)
+
+React Native date & time picker component for iOS and Android
+
+## Getting started
+
+`npm install react-native-datetimepicker --save`
+
+or
+
+`yarn add react-native-datetimepicker`
+
+
+### Using React Native Link
+
+`react-native link react-native-datetimepicker`
+
+### Manual installation
+
+#### iOS
+
+- Linking project manually:
+  
+  1. In the XCode's "Project navigator", right click on your project's Libraries folder ➜ `Add Files to <...>`.
+  2. Go to `node_modules` ➜ `react-native-datetimepicker` ➜ `ios` ➜ select `RNDateTimePicker.xcodeproj`.
+  3. Add `libRNDateTimePicker.a` to `Build Phases -> Link Binary With Libraries`.
+
+- Using CocoaPods:
+
+  1. Install CocoaPods, here the [installation guide](https://guides.cocoapods.org/using/getting-started.html).
+  2. Inside the iOS folder run `pod init`, this will create the initial `pod` file.
+  3. Update your `pod` file to look like the following ( Remember to replace `MyApp` with your target name ):
+   
+      ```ruby
+      # Allowed sources
+      source 'https://github.com/CocoaPods/Specs.git'
+
+      target 'MyApp' do
+        # As we use Swift, ensure that `use_frameworks` is enabled.
+        use_frameworks!
+
+        # Specific iOS platform we are targetting
+        platform :ios, '8.0'
+
+        # Point to the installed version
+        pod 'RNDateTimePicker', :path => '../node_modules/react-native-datetimepicker/RNDateTimePicker.podspec'
+
+        # React/React-Native specific pods
+        pod 'React', :path => '../node_modules/react-native', :subspecs => [
+          'Core',
+          'CxxBridge',      # Include this for RN >= 0.47
+          'DevSupport',     # Include this to enable In-App Devmenu if RN >= 0.43
+          'RCTText',
+          'RCTNetwork',
+          'RCTWebSocket',   # Needed for debugging
+        ]
+
+        # Explicitly include Yoga if you are using RN >= 0.42.0
+        pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+
+        # Third party deps podspec link
+        pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+        pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+        pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+      end
+      ```
+  4. Run `pod install` inside the same folder where the `pod` file was created
+  5. Open the `.xcworkspace` file that was created and run
```

Comment:
run what? Also why go through xcode, the norm is using the `react-native` CLI, e.g. `react-native run-{ios,android}`

Reply:
run from Xcode, I will change it to use CLI.